docs/cli.md

```diff
@@ -368,6 +368,220 @@ Output will look like this, where `python` will be green as it is the package th
+│   ├── pyyaml v6.0.1
+│   │   ├── python_abi v3.12
+│   │   ├── python v3.12.2 (*)
+│   │   └── yaml v0.2.5
+│   ├── identify v2.5.35
+│   │   └── python v3.12.2 (*)
+│   ├── python v3.12.2 (*)
+│   ├── virtualenv v20.25.1
+│   │   ├── distlib v0.3.8
+│   │   │   └── python v3.12.2 (*)
+│   │   ├── python v3.12.2 (*)
+│   │   ├── filelock v3.13.1
+│   │   │   └── python v3.12.2 (*)
+│   │   └── platformdirs v4.2.0
+│   │       └── python v3.12.2 (*)
+│   └── nodeenv v1.8.0
+│       └── python v3.12.2 (*)
+├── rust v1.76.0
+│   └── rust-std-aarch64-apple-darwin v1.76.0
+├── openssl v3.2.1
+├── pkg-config v0.29.2
+│   ├── libglib v2.78.4
+│   │   ├── libiconv v1.17
+│   │   ├── gettext v0.21.1
+│   │   │   └── libiconv v1.17 (*)
+│   │   ├── libffi v3.4.2 (*)
+│   │   ├── libcxx v16.0.6
+│   │   ├── pcre2 v10.42
+│   │   │   ├── libzlib v1.2.13 (*)
+│   │   │   └── bzip2 v1.0.8 (*)
+│   │   └── libzlib v1.2.13 (*)
+│   └── libiconv v1.17 (*)
+├── git v2.42.0
+│   ├── libexpat v2.6.2 (*)
+│   ├── libzlib v1.2.13 (*)
+│   ├── perl v5.32.1
+│   ├── pcre2 v10.42 (*)
+│   ├── openssl v3.2.1 (*)
+│   └── libiconv v1.17 (*)
+├── cffconvert v2.0.0
+│   ├── ruamel.yaml v0.18.6
+│   │   ├── python_abi v3.12 (*)
+│   │   ├── ruamel.yaml.clib v0.2.8
+│   │   │   ├── python_abi v3.12 (*)
+│   │   │   └── python v3.12.2 (*)
+│   │   └── python v3.12.2 (*)
+│   ├── jsonschema v3.2.0
+│   │   ├── python v3.12.2 (*)
+│   │   ├── pyrsistent v0.20.0
+│   │   │   ├── python_abi v3.12 (*)
+│   │   │   └── python v3.12.2 (*)
+│   │   ├── six v1.16.0
+│   │   └── attrs v23.2.0
+│   │       └── python v3.12.2 (*)
+│   ├── requests v2.31.0
+│   │   ├── idna v3.6
+│   │   │   └── python v3.12.2 (*)
+│   │   ├── python v3.12.2 (*)
+│   │   ├── urllib3 v2.2.1
+│   │   │   ├── pysocks v1.7.1
+│   │   │   │   └── python v3.12.2 (*)
+│   │   │   ├── python v3.12.2 (*)
+│   │   │   └── brotli-python v1.1.0
+│   │   │       ├── python v3.12.2 (*)
+│   │   │       ├── python_abi v3.12 (*)
+│   │   │       └── libcxx v16.0.6 (*)
+│   │   ├── certifi v2024.2.2
+│   │   │   └── python v3.12.2 (*)
+│   │   └── charset-normalizer v3.3.2
+│   │       └── python v3.12.2 (*)
+│   ├── click v8.1.7
+│   │   └── python v3.12.2 (*)
+│   ├── pykwalify v1.8.0
+│   │   ├── python v3.12.2 (*)
+│   │   ├── docopt v0.6.2
+│   │   ├── python-dateutil v2.9.0
+│   │   │   ├── python v3.12.2 (*)
+│   │   │   └── six v1.16.0 (*)
+│   │   └── ruamel.yaml v0.18.6 (*)
+│   └── python v3.12.2 (*)
+└── tbump v6.9.0
+    ├── cli-ui v0.17.2
+    │   ├── colorama v0.4.6
+    │   │   └── python v3.12.2 (*)
+    │   ├── python v3.12.2 (*)
+    │   ├── tabulate v0.9.0
+    │   │   └── python v3.12.2 (*)
+    │   └── unidecode v1.3.8
+    │       └── python v3.12.2 (*)
+    ├── python v3.12.2 (*)
+    ├── schema v0.7.5
+    │   ├── contextlib2 v21.6.0
+    │   │   └── python v3.12.2 (*)
+    │   └── python v3.12.2 (*)
+    ├── tomlkit v0.12.4
+    │   └── python v3.12.2 (*)
+    └── docopt v0.6.2 (*)
+```
```

Comment:
I think the current ~120 lines could be shrunk to 20. As here it would just give an idea to the user what it could look like. 

Showing interesting things like the 4 layers deep, the `(*)` and the